core/tx_pool.go

```diff
@@ -1254,6 +1282,7 @@ func (as *accountSet) flatten() []common.Address {
 type txLookup struct {
 	all  map[common.Hash]*types.Transaction
 	lock sync.RWMutex
+	totalSize common.StorageSize
```

Comment:
Please rename to `memory` and move one line up. The lock is an irrelevant maintenance field that should be the last one.